docs/api-reference/next.config.js/runtime-configuration.md

```diff
@@ -53,7 +53,9 @@ function MyImage() {
   )
 }
 
-export default MyImage
+ImagePage.getInitialProps = async ({}) => ({});
```

Comment:
`publicRuntimeConfig` should be available regardless of the page being static or not so I don't think we want to make this change to the doc

Reply:
True, sorry for the oversight . Thank you.